src/funnels/automation/StableSwapper.sol

```diff
@@ -51,41 +48,33 @@ contract StableSwapper {
         _;
     }
 
-    // permissionned to facilitators
-    modifier toll { 
-        require(buds[msg.sender] == 1, "StableSwapper/non-facilitator"); 
-        _;
-    }
-
     // permissionned to whitelisted keepers
-    modifier keep { 
-        require(bots[msg.sender] == 1, "StableSwapper/non-keeper"); 
+    modifier toll { 
+        require(buds[msg.sender] == 1, "StableSwapper/non-keeper");
         _;
     }
 
     uint256 internal constant WAD = 10 ** 18;
 
     function rely  (address usr) external auth { wards[usr] = 1; emit Rely(usr); }
     function deny  (address usr) external auth { wards[usr] = 0; emit Deny(usr); }
-    function kiss  (address usr) external auth {  buds[usr] = 1; emit Kissed(usr); }
-    function diss  (address usr) external auth {  buds[usr] = 0; emit Dissed(usr); }
-    function permit(address usr) external toll {  bots[usr] = 1; emit Permit(usr); }
-    function forbid(address usr) external toll {  bots[usr] = 0; emit Forbid(usr); }
+    function kiss  (address usr) external auth {  buds[usr] = 1; emit Kiss(usr); }
+    function diss  (address usr) external auth {  buds[usr] = 0; emit Diss(usr); }
```

Comment:
Let's also set them to multi line and remove the space before the parenthesis :)

Reply:
Or well maybe this is part of the other PR, got a bit confused we are changing StableSwapper here.